src/js/pages/Index.js

```diff
@@ -60,17 +58,14 @@ var Index = React.createClass({
         suppressUpdate: true
       }
     ];
-
-    const state = getSidebarState();
-    this.internalStorage_set(state);
   },
 
   componentDidMount() {
     SidebarStore.addChangeListener(
       EventTypes.SIDEBAR_CHANGE,
       this.onSideBarChange
     );
-    global.addEventListener("resize", this.handleWindowResize);
+    global.addEventListener("resize", this.handleWindowResize.bind(this));
```

Comment:
I am getting an error here.
![image](https://user-images.githubusercontent.com/156010/43326136-561a2590-91b8-11e8-9d44-297ef272ae0b.png)

Reply:
This is an warning not a error BUT I removed the bind anyways because I'm removing the listener so no need to bind.